packages/adapters/storage/src/test_instance.rs

```diff
@@ -232,13 +235,18 @@ impl block_importer::port::Storage for DbWithProcess {
 }
 
 impl block_bundler::port::Storage for DbWithProcess {
-    async fn lowest_sequence_of_unbundled_blocks(
+    async fn next_candidates_for_bundling(
         &self,
         starting_height: u32,
-        max_cumulative_bytes: u32,
+        max_cumulative_bytes: BytesLimit,
+        block_buildup_threshold: u32,
     ) -> services::Result<Option<UnbundledBlocks>> {
         self.db
-            ._lowest_unbundled_blocks(starting_height, max_cumulative_bytes)
+            ._next_candidates_for_bundling(
```

Comment:
[nit] In `fuel-core`, we try not to use `_` for internal methods. Instead, we add `_inner` suffix.